skaha/src/main/java/org/opencadc/skaha/SkahaAction.java

```diff
@@ -282,9 +282,11 @@ public Image getImage(String imageID) throws Exception {
                         if (!jArtifact.isNull("labels")) {
                             String digest = jArtifact.getString("digest");
                             JSONArray labels = jArtifact.getJSONArray("labels");
-                            String type = getTypeFromLabels(labels);
-                            if (type != null) {
-                                return new Image(imageID, type, digest);
+                            Set<String> types = getTypesFromLabels(labels);
+                            if (types.size() > 0) {
+                                // TODO: fix the cardinality of types to image.
+                                // ie--A running image has 1 type, but an image can have multiple
```

Comment:
"but an image can have multiple" something. Looks incomplete. Is it "types"?

Reply:
Thanks, I've improved the English